ui/raidboss/data/triggers/o4n.js

```diff
@@ -0,0 +1,145 @@
+'use strict';
+
+// O4n - Deltascape 4.0 Normal
+
+[{
+
+  zoneRegex: /Deltascape \(V4.0\)/,
+  timelineFile: 'o4n.txt',
+  triggers: [
+    {
+      id: 'O4n Doom',
+      regex: / 1A:........:(\y{Name}) gains the effect of Doom/,
+      condition: function(data) {
+        return data.canCleanse();
+      },
+      alertText: function(data, matches) {
+        return {
+          en: 'Cleanse ' + data.shortName(matches[1]),
```

Comment:
I forget how this fight works, are there multiple dooms at once?

Reply:
Eep, you're correct. This will notify twice, once for each Doom. I'll change that over to notify to "cleanse soon", with suppression if necessary.